src/RoadMarking/RoadMarking.tsx

```diff
@@ -0,0 +1,56 @@
+import { Line } from "react-konva";
+import React from "react";
+import { RoadMarkingProps } from "./RoadMarking.types";
+
+const RoadMarking = (
+  {
+    points,
+    dash = [],
+    width = 0.15,
+    type = "single",
+    color = "#ffffff"
+  }: RoadMarkingProps,
+  key: React.Key
+) => {
+  const pointsXY = points.filter(
+    (_, index) => index % 3 === 0 || (index - 1) % 3 === 0
+  );
+  switch (type) {
+    case "double":
+      // plot symmetrical double lines
+      return (
+        <>
+          <Line
+            key={key}
+            strokeWidth={width * 2.5}
+            scaleY={-1}
+            dash={dash}
+            stroke={color}
+            points={pointsXY}
+          />
+          <Line
+            key={`${key}_cutout`}
+            strokeWidth={width * 0.5}
+            scaleY={-1}
+            stroke={color}
+            globalCompositeOperation="destination-out"
```

Comment:
I think this part requires some investigation. In storybook, it seems to me that the cutout is not transparent, but somewhere between the background color and the line's color. It can get difficult to notice e.g. in case of yellow line, see the posted image. If this is a problem, I would suggest to add another, optional prop "cutoutColor" (or something similar), which could define the color of the cutout.
![yellowdoubleline](https://github.com/IPG-Automotive-UK/react-ui/assets/163292068/6302b88e-05cf-453c-a3ae-3b6b978b1835)

Reply:
Seems like it is to do with the size of the component. I think it is only a display error/funny thing. I have tried both increasing the `width` of the line and increase the `scaling` of the stage and both of these methods resolve the weird shadow in the cutout. BTW yellow, is probably not the greatest color cause I can barely see the picture.

Anyways, at some size the display is proper. Thus the `global-composition` is working. As you can zoom in and out on the road viewer this should not be a huge issues. I think it is best to accept this behavior for now.
![image](https://github.com/IPG-Automotive-UK/react-ui/assets/67595415/4c04d921-f4fd-4971-9dd4-9ccffcc0e9be)

Reply:
For now I have increased the `scaling` on the stage, so the visual issue should not be visible on the default component